lib_eio_posix/low_level.ml

```diff
@@ -60,10 +60,11 @@ let rec connect fd addr =
   with
   | Unix.Unix_error (EINTR, _, _) -> connect fd addr   (* Just in case *)
```

Comment:
This EINTR should be handled like EINPROGRESS below, and I think you can zap EAGAIN | EWOULDBLOCK | EALREADY